src/event-bus.js

```diff
@@ -0,0 +1,4 @@
+import Vue from 'vue';
```

Comment:
Why do we need to pass data from children to parent? React Flux actually limits data flow to one direction to reduce the amount of complexity